util/libcrypto.num

```diff
@@ -5043,14 +5043,14 @@ OSSL_PARAM_set_all_unmodified           5170	3_0_0	EXIST::FUNCTION:
 EVP_RAND_fetch                          5171	3_0_0	EXIST::FUNCTION:
 EVP_RAND_up_ref                         5172	3_0_0	EXIST::FUNCTION:
 EVP_RAND_free                           5173	3_0_0	EXIST::FUNCTION:
-EVP_RAND_number                         5174	3_0_0	EXIST::FUNCTION:
-EVP_RAND_name                           5175	3_0_0	EXIST::FUNCTION:
+EVP_RAND_get_number                     5174	3_0_0	EXIST::FUNCTION:
+EVP_RAND_get0_name                      5175	3_0_0	EXIST::FUNCTION:
 EVP_RAND_is_a                           5176	3_0_0	EXIST::FUNCTION:
-EVP_RAND_provider                       5177	3_0_0	EXIST::FUNCTION:
+EVP_RAND_get0_provider                  5177	3_0_0	EXIST::FUNCTION:
 EVP_RAND_get_params                     5178	3_0_0	EXIST::FUNCTION:
 EVP_RAND_CTX_new                        5179	3_0_0	EXIST::FUNCTION:
 EVP_RAND_CTX_free                       5180	3_0_0	EXIST::FUNCTION:
-EVP_RAND_CTX_rand                       5181	3_0_0	EXIST::FUNCTION:
+EVP_RAND_CTX_get0_rand                  5181	3_0_0	EXIST::FUNCTION:
 EVP_RAND_CTX_get_params                 5182	3_0_0	EXIST::FUNCTION:
```

Comment:
Should the get_params & set_params calls be changed also?

Reply:
Changed to what specifically?

Reply:
get0 ?

Reply:
But these aren't doing a get0.  They copy the data into the params, so get1 is more appropriate.  Except for PTR types where there is no copy...

An argument could be made for the gettable/settable calls to include a 0 but that's a different matter.

Reply:
I do not think the get_/set_params should be renamed. The semantics is not a clear get1/get0. Anyway - a different PR if OTC decides to rename them.

Reply:
Not a problem - just thought I would mention it..